integration/test_project_management.py

```diff
@@ -14,14 +14,24 @@
 
```

Comment:
Can you also start cranking out some unit tests? Note that we only run integration tests during releases. We need a set of fast and extensive unit tests, that we run during development and on our CI servers.

Reply:
Please see the following pull requests #218 + #219.